src/ripple/basics/Expected.h

```diff
@@ -132,17 +135,16 @@ class [[nodiscard]] Expected
     using Base = boost::outcome_v2::result<T, E, detail::throw_policy>;
 
 public:
-    template <
-        typename U,
-        typename = std::enable_if_t<std::is_convertible_v<U, T>>>
-    constexpr Expected(U r) : Base(T{std::forward<U>(r)})
+    template <typename U>
+    requires std::convertible_to<U, T> constexpr Expected(U && r)
```

Comment:
This is because we use clang 10 for formatting. When I use clang 15 is formats in a reasonable way. I hope we can update our clang format shortly. I also don't mind delaying this patch until after we update formatting.